index.html

```diff
@@ -4,12 +4,13 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Santiago & George Portfolio</title>
     <link rel="stylesheet" href="styles.css">
 </head> 
 <body>
     <header class="bar-container">
         <nav>
+            <img src="./Images/app_bar.png" alt="bar">
```

Comment:
Nice job so far👍. For decorative images or images or images that are not important for understanding the content of the webpage, the alt attribute should be given a null value `alt=" "` a value of a space😉. This makes the screen reader ignore the image😉. you can make reference to this [lesson](https://www.w3.org/WAI/test-evaluate/preliminary/#images) for clarity😉.

<img width="712" alt="image" src="https://github.com/Arch-Noize/portfolio-microverse/assets/64643995/e56a0e95-6d39-4e8a-af24-92a25fae57be">